src/login/styles/_buttons.scss

```diff
@@ -95,10 +95,6 @@ button.close:hover {
   transform: scale(1.2);
 }
 
-button.btn#mobile-button {
-  margin-top: 0;
-}
-
 //  {
```

Comment:
please delete this code if not in use for general clean up

Reply:
this has been already removed